code/Forms/SiteTreeURLSegmentField.php

```diff
@@ -68,7 +68,7 @@ public function Field($properties = [])
      * @param HTTPRequest $request
      * @return string
      */
-    public function suggest($request)
+    public function suggest(HTTPRequest $request)
```

Comment:
Is it okay that this doesn't return `HTTPResponse`?

Reply:
It returns a string. Probably shouldn't have even bothered adding strong typing to the param here, though harmless to leave as is